.github/workflows/codecov_ats.yml

```diff
@@ -80,13 +83,36 @@ jobs:
           env:
             CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
             CODECOV_STATIC_TOKEN: ${{ secrets.CODECOV_STATIC_TOKEN }}
-        # Run label analysis IN DRY MODE (no tests will actually run)
+        # Run label analysis in dry mode to get the list of tests to run
+        # The base commit will be the parent commit (apparently commits on master don't exist in codecov)
         - name: Label Analysis
```

Comment:
Can you add the word "codecov" in here somewhere?
"label analysis" is more commonly known as "automatic test selection", I think?

low priority